types/mac.go

```diff
@@ -143,24 +143,28 @@ func (a *MACArray) Value(i int) net.HardwareAddr {
 	if a.IsNull(i) {
 		return nil
 	}
-	return net.HardwareAddr(a.Storage().(*array.Binary).Value(i))
```

Comment:
Here Binary is more suitable as well

Reply:
why? it is always string representable and in most places it is stored at string.

Reply:
Hm, with this I'm actually OK to go in any direction (as the other platforms/tools might not guarantee the same byte layout).